great_expectations/expectations/expectation.py

```diff
@@ -694,6 +698,206 @@ def build_configuration(cls, *args, **kwargs):
             meta=meta,
         )
 
+    def self_check(self, pretty_print=True):
+
+        camel_name = self.__class__.__name__
+        snake_name = camel_to_snake(self.__class__.__name__)
+        docstring, short_description = self._get_docstring_and_short_description()
+        library_metadata = self._get_library_metadata()
```

Comment:
Similarly, library metadata should be moved out of the class instance into the module.

Reply:
Keeping this in for the same reasons as examples...

Reply:
Resolving this conversation, since we agreed since to keep it in the class.